jvm/src/test/scala/uk/gov/homeoffice/drt/protobuf/serialisation/FlightMessageConversionSpec.scala

```diff
@@ -163,4 +195,55 @@ class FlightMessageConversionSpec extends Specification {
       }
     }
   }
+
+  "when flight message is deserialize and if there is apiPax present then arrival should have apiPax in totalPax" >> {
+    val apiFlight = arrival.copy(
+      FeedSources = Set(ApiFeedSource),
+      PassengerSources = Map(ApiFeedSource -> Passengers(Option(95), None),
+      ))
+
+    val flightMessage = getFlightMessageWithoutPax(apiFlight).copy(
+      feedSources = Seq(ApiFeedSource.toString),
+      apiPax = apiFlight.PassengerSources.get(ApiFeedSource).flatMap(_.getPcpPax)
+    )
+
+    val arrivalResult = FlightMessageConversion.flightMessageToApiFlight(flightMessage)
+
+    arrivalResult === apiFlight
+  }
+
+  "when flight message is deserialize and if there is actPax and/or transPax present then arrival should have actPax and transPax in totalPax" >> {
```

Comment:
Good tests :+1: 
A couple of scenarios we should add:
- 1. An arrival message has pax sources and different actpax & tranpax values - ie these should be ignored not be added to the list of sources
- 2. An arrival message has act & tran pax and eg acl & port forecast sources - test that the correct source is used for the migration